arenadata/Dockerfile.ubuntu

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:22.04 as base
+
+ARG sigar=https://downloads.adsw.io/ADB/6.27.1_arenadata56/ubuntu/22.04/community/x86_64/packages/sigar_1.6.5-3304%2Bgite8961a6_all.deb
+ARG sigar_headers=https://downloads.adsw.io/ADB/6.27.1_arenadata56/ubuntu/22.04/community/x86_64/packages/sigar-headers_1.6.5-3304%2Bgite8961a6_all.deb
+ARG libxerces=https://downloads.adsw.io/ADB/6.27.1_arenadata56/ubuntu/22.04/community/x86_64/packages/libxerces-c31_3.1.2-3304_amd64.deb
+ARG libxerces_dev=https://downloads.adsw.io/ADB/6.27.1_arenadata56/ubuntu/22.04/community/x86_64/packages/libxerces-c31-dev_3.1.2-3304_amd64.deb
+
+WORKDIR /home/gpadmin
+
+COPY README.ubuntu.bash ./
+RUN ./README.ubuntu.bash && \
+    rm README.ubuntu.bash && \
```

Comment:
centos's docker file does not contain these steps

Reply:
The current approach is better than approach used in centos's docker file, because we keep package list to install in one place